### PR TITLE
Python demos: performance metrics unification

### PR DESCRIPTION
**Pull request by anzhella-pankratova**
Added performance metrics measurement to demos: 
- FPS only to demos with supporting of video processing
- latency to all demos. The latency is the total processing time, starting from reading the data to the results displaying (in cmd either in cv2 window)

### CODE REVIEW

File: demos/noise_suppression_demo/python/noise_suppression_demo.py [hunk 1]
```diff
@@ -134,16 +135,11 @@ def main():
...
+        total_latency += perf_counter() - start_time
         samples_out.append(res["output"].buffer.squeeze(0))
 
-    log.info("Sequence of length {:0.2f}s is processed by {:0.2f}s".format(
-        sum(s.shape[0] for s in samples_out)/16000,
-        sum(samples_times),
-
-    ))
+    log.info("Metrics report:")
+    log.info("\tLatency: {:.1f} ms".format(total_latency * 1e3))
```
Comment (eizamaliev) [inline demos/noise_suppression_demo/python/noise_suppression_demo.py:142]: @Wovchena @vladimir-dudnik Here we got ambiguous metric:
Here and before latency is calculated as a total inference time. But to process an audio demo runs model several time, it depends on model input size and audio length. So what is more interesting - time of single inference or  whole time of audio processing?

Comment (vladimir-dudnik) [reply demos/noise_suppression_demo/python/noise_suppression_demo.py:142]: in openvino [FAQ](https://docs.openvinotoolkit.org/latest/openvino_docs_performance_benchmarks_faq.html) latency is defined for synchronous execution mode, and in DL WorkBench [docs](https://docs.openvinotoolkit.org/latest/workbench_docs_Workbench_DG_Key_Concepts.html) it is additionally notes, that latency is measured for single unit of work. Throughput metric, in contrast, measured as number of units of work processed per defined period of time (for example, frames per second).

**Review by Wovchena — CHANGES_REQUESTED** [review]

File: demos/noise_suppression_demo/python/noise_suppression_demo.py (hunk 1 above)
Comment (Wovchena) [reply demos/noise_suppression_demo/python/noise_suppression_demo.py:142]: My perception was that we agreed on what we measure two weeks ago and this PR follows this. Moreover README clearly defines: `**Latency**: total processing time required to process input data (from reading the data to displaying the results).`, so there is no ambiguity.

File: demos/colorization_demo/python/colorization_demo.py [hunk 2]
```diff
@@ -87,6 +89,7 @@ def build_arg():
 
     _, _, h_in, w_in = input_shape
 
+    start_time = perf_counter()
```
[the remark this answers is not in the document]
Comment (anzhella-pankratova) [reply demos/colorization_demo/python/colorization_demo.py]: Put the frame reading after the initialization of variables where it is possible. However, `video_writer` opening requires `frame.shape`, so it cannot be moved.

File: demos/handwritten_text_recognition_demo/python/handwritten_text_recognition_demo.py [hunk 3]
```diff
@@ -104,15 +106,19 @@ def main():
...
     for i in range(args.number_iter):
-        t0 = time.time()
+        start_time = perf_counter()
         preds = exec_net.infer(inputs={input_blob: input_image})
         preds = preds[out_blob]
         result = codec.decode(preds)
         print(result)
-        infer_time.append((time.time() - t0) * 1000)
-    log.info("Average throughput: {} ms".format(np.average(np.asarray(infer_time))))
+        total_latency += perf_counter() - start_time
```
[the remark this answers is not in the document]
Comment (anzhella-pankratova) [reply demos/handwritten_text_recognition_demo/python/handwritten_text_recognition_demo.py]: Updated

File: demos/image_inpainting_demo/python/image_inpainting_demo.py [hunk 4]
```diff
@@ -106,10 +108,12 @@ def inpaint_auto(img, inpainting_processor, args):
     #--- Inpaint and show results
     output_image = inpainting_processor.process(masked_image, mask)
     concat_imgs = np.hstack((masked_image, output_image))
-    cv2.putText(concat_imgs, 'Performance: {:.1f} FPS'.format(float(1 / inpainting_processor.infer_time)), (5, 15), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 200))
     cv2.putText(concat_imgs, 'original', (5, 15), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 100))
-    cv2.putText(concat_imgs, 'result', (concat_imgs.shape[1]-5-cv2.getTextSize('result', cv2.FONT_HERSHEY_COMPLEX, 0.5, 1)[0][0], 15), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 100))
-    cv2.putText(concat_imgs, 'summary: {:.1f} FPS'.format(float(1 / inpainting_processor.infer_time)), (5, 35), cv2.FONT_HERSHEY_COMPLEX, 0.5, (0, 0, 200))
```
[the remark this answers is not in the document]
Comment (anzhella-pankratova) [reply demos/image_inpainting_demo/python/image_inpainting_demo.py:112]: Moved end_time measurement before `cv2.putText()` and added latency showing on the image.

File: demos/noise_suppression_demo/python/noise_suppression_demo.py [hunk 5]
```diff
@@ -134,16 +135,11 @@ def main():
         infer_request_ptr.infer()
         res = infer_request_ptr.output_blobs
 
-        t1 = time.perf_counter()
-
-        samples_times.append(t1-t0)
+        total_latency += perf_counter() - start_time
```
[the remark this answers is not in the document]
Comment (anzhella-pankratova) [reply demos/noise_suppression_demo/python/noise_suppression_demo.py]: Updated

**Review by Wovchena — APPROVED** [review]

File: demos/noise_suppression_demo/python/noise_suppression_demo.py (hunk 1 above)
Comment (vladimir-dudnik) [reply demos/noise_suppression_demo/python/noise_suppression_demo.py]: ok, then

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**